### PR TITLE
SC - 6657 - improve naming methods

### PR DESCRIPTION
**Pull request by Grzesiek80**
# Description
improve naming methods

## Links to Tickets or other pull requests
https://ticketsystem.hpi-schul-cloud.org/browse/SC-6657

## Changes
change name of methods

### CODE REVIEW

File: page-objects/pages/HMWRKHomeworkListPage.js [hunk 1]
```diff
@@ -99,7 +99,7 @@ module.exports = {
         return isTaskOnList;
     },
 
-    userFindsTheTask: async function (taskname) {
+    getTaskFromList: async function (taskname) {
```
Comment (alstanki) [inline page-objects/pages/HMWRKHomeworkListPage.js:102]: It is not "get"  its click (there is no return). Maybe I am wrong but it is not a duplicate of  clickOnTask(). But this method looks better. It has more checks.

File: page-objects/pages/HMWRKHomeworkPage.js [hunk 2]
```diff
@@ -40,7 +40,7 @@ module.exports = {
 
 
     studentEditsTextHomeworkAndSubmits: async function () {
```
Comment (alstanki) [inline page-objects/pages/HMWRKHomeworkPage.js:42]: I do not see a student editing anything here. The second thing is, should userRole be written in the name?

File: page-objects/pages/HMWRKHomeworkPage.js [hunk 3]
```diff
@@ -40,7 +40,7 @@ module.exports = {
...
-        await this.switchToSubmissionTab();
+        await this.clickOnSubmissionTab();
         await this.submitSolutionForTheHometask();
     },
     // teacher helpers
```
Comment (alstanki) [inline page-objects/pages/HMWRKHomeworkPage.js:46]: again . Do we need userRole Here? hasTheStudentSubmittedTheTask -> Shouldn't it look like this?  isTaskSubmitted

**Review by yuliashikhareva — DISMISSED** [review]
I think the naming is good.

File: page-objects/pages/NWSNewsListPage.js [hunk 4]
```diff
@@ -11,18 +11,18 @@ module.exports = {
 	goToNews: async function() {
 		await elementHelpers.loadPage(url, 100);
     },
-    verifyWhetherVisible: async function() {
+    getAllNews: async function() {
```
Comment (alstanki) [inline page-objects/pages/NWSNewsListPage.js:14]: getListOfNewNames

File: page-objects/pages/coursePages/CRSSCourseListPage.js [hunk 5]
```diff
@@ -115,11 +115,11 @@ function getColourSelector(colourName) {
 	return colourSelector;
 }
 
-async function fillCourseNameIntoSearchInputField(courseName) {
+async function setCourseNameIntoSearchInputField(courseName) {
 	await elementHelpers.fillInputField(searchCourseFiled, courseName);
 }
 
-async function countDisplayedCoursesForSection(section) {
+async function getNumberOfDisplayedCoursesForSection(section) {
```
Comment (alstanki) [inline page-objects/pages/coursePages/CRSSCourseListPage.js:122]:
```suggestion
async function getCountOfDisplayedCoursesForSection(section) {
```

File: page-objects/pages/coursePages/CRSSCourseListPage.js [hunk 6]
```diff
@@ -226,7 +226,7 @@ async function getListOfCourseTitlesInSection(section) {
 	return courseTitleList;
 }
 
-async function countCoursesWhichTitlesContainTextInSection(text, section) {
+async function getNumberOfCoursesWhichTitlesContainTextInSection(text, section) {
```
Comment (alstanki) [inline page-objects/pages/coursePages/CRSSCourseListPage.js:229]:
```suggestion
async function getCountOfCoursesWhichTitlesContainTextForSection(text, section) {
```

File: page-objects/pages/coursePages/CRSSCourseListPage.js [hunk 7]
```diff
@@ -272,15 +272,15 @@ async function studentLogsInAndGoesToTasksOfTheCourse(username, password, course
 	await goToTasksOfTheCourse(coursename);
 }
 
-async function verifyCourseAndTopic(coursename, topicname, section) {
+async function isTopicInCourse(coursename, topicname, section) {
```
Comment (alstanki) [inline page-objects/pages/coursePages/CRSSCourseListPage.js:275]:
```suggestion
async function isTopicInCourseInSection(coursename, topicname, section) {
```

File: page-objects/pages/coursePages/CRSSCourseListPage.js [hunk 8]
```diff
@@ -272,15 +272,15 @@ async function studentLogsInAndGoesToTasksOfTheCourse(username, password, course
...
 	await clickOnCourseInSection(coursename, section);
 	let topicNames = await Promise.all(
 		(await driver.$$('#topic-list > div > div > div')).map(async (element) => await element.getText())
 	);
 	await expect(topicNames).to.include(topicname);
 }
 
-async function verifyCopyWithStudents(coursename) {
+async function areStudentsInCopiedCourse(coursename) {
```
Comment (alstanki) [inline page-objects/pages/coursePages/CRSSCourseListPage.js:283]: The method checks that there are NO students in copied course

File: page-objects/pages/teamsPages/TMSTeamMembersPage.js [hunk 9]
```diff
@@ -30,10 +30,10 @@ module.exports = {
 	addTwoTeamMemebers: async function (teammember1, teammember2) {
 		await this.addTeamMembersSteps(teammember1);
 		await this.addTeamMembersSteps(teammember2);
-		await this.submitAddTeammemberAfterAllMemebersWereAdded();
+		await this.submitAddTeamMemberAfterAllMemebersWereAdded();
 		await driver.pause(1500);
 	},
-	submitAddTeammemberAfterAllMemebersWereAdded: async function () {
+	submitAddTeamMemberAfterAllMemebersWereAdded: async function () {
```
Comment (alstanki) [inline page-objects/pages/teamsPages/TMSTeamMembersPage.js:36]:
```suggestion
	clickSubmitAddTeamMember: async function () {
```
AfterAllMemebersWereAdded -> it is not needed
its click method

File: page-objects/pages/DashboardPage.js [hunk 10]
```diff
@@ -19,24 +19,24 @@ module.exports = {
...
 	},
 
-	loginResultDashboard: async function () {
+	isTitleOfDashboard: async function () {
 		await this.goToDashboard();
 		expect(await elementHelpers.getElementText(dashboardHeader)).to.equal(dashboardTitle);
 	},
 
-	loginInitials: async function () {
+	areInitialsCorrect: async function () {
```
Comment (alstanki) [inline page-objects/pages/DashboardPage.js:27]:
```suggestion
	areUserInitialsCorrect: async function () {
```

**Review by alstanki — CHANGES_REQUESTED** [review]

File: page-objects/pages/HMWRKHomeworkPage.js (hunk 2 above)
Comment (yuliashikhareva) [reply page-objects/pages/HMWRKHomeworkPage.js:42]: this method will be deleted (refactored). It was a short time solution from @Grzesiek80 , I think it is good.

File: page-objects/pages/HMWRKHomeworkPage.js (hunk 3 above)
Comment (yuliashikhareva) [reply page-objects/pages/HMWRKHomeworkPage.js:46]: the same here as above

File: page-objects/pages/DashboardPage.js (hunk 10 above)
Comment (Grzesiek80) [reply page-objects/pages/DashboardPage.js:27]: done

File: page-objects/pages/teamsPages/TMSTeamMembersPage.js (hunk 9 above)
Comment (Grzesiek80) [reply page-objects/pages/teamsPages/TMSTeamMembersPage.js:36]: done

File: page-objects/pages/coursePages/CRSSCourseListPage.js (hunk 8 above)
Comment (Grzesiek80) [reply page-objects/pages/coursePages/CRSSCourseListPage.js:283]: done

File: page-objects/pages/coursePages/CRSSCourseListPage.js (hunk 7 above)
Comment (Grzesiek80) [reply page-objects/pages/coursePages/CRSSCourseListPage.js:275]: done

File: page-objects/pages/coursePages/CRSSCourseListPage.js (hunk 6 above)
Comment (Grzesiek80) [reply page-objects/pages/coursePages/CRSSCourseListPage.js:229]: done

File: page-objects/pages/coursePages/CRSSCourseListPage.js (hunk 5 above)
Comment (Grzesiek80) [reply page-objects/pages/coursePages/CRSSCourseListPage.js:122]: done

File: page-objects/pages/NWSNewsListPage.js (hunk 4 above)
Comment (Grzesiek80) [reply page-objects/pages/NWSNewsListPage.js:14]: done

File: page-objects/pages/HMWRKHomeworkPage.js (hunk 3 above)
Comment (Grzesiek80) [reply page-objects/pages/HMWRKHomeworkPage.js:46]: will be refactored

File: page-objects/pages/HMWRKHomeworkPage.js (hunk 2 above)
Comment (Grzesiek80) [reply page-objects/pages/HMWRKHomeworkPage.js:42]: will be refactored

File: page-objects/pages/HMWRKHomeworkListPage.js (hunk 1 above)
Comment (Grzesiek80) [reply page-objects/pages/HMWRKHomeworkListPage.js:102]: there is a diffrent method but renamed it. done

**Review by alstanki — APPROVED** [review]

**Pull request merged**